lib/msf/base/sessions/ldap.rb

```diff
@@ -14,18 +14,30 @@ class Msf::Sessions::LDAP
   # @return [Rex::Proto::LDAP::Client] The LDAP client
   attr_accessor :client
 
+  attr_accessor :keep_alive_thread
```

Comment:
NAB: We've got a mix of `keep_alive` and `keepalive` let's pick one (I'd say `keep_alive` since that seems more dominant in the existing winrm session) and keep it consistent